.changeset/brave-grapes-provide.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/integration': minor
+'@backstage/plugin-catalog-backend': minor
```

Comment:
Don't think we need a minor bump of the catalog backend, nothing breaking there right?

I think it could actually be nice to split this into two changesets as well, with each of them explaining the change from each package's point of view